src/features/stats.ts

```diff
@@ -32,19 +29,38 @@ const emitEvent = (
   fetch(`https://api.amplitude.com/httpapi?${queryString.stringify(fields)}`);
 };
 
-const EVENTS = {
-  message: "message sent",
-  newMember: "new member joined",
-  memberLeft: "member left server",
+const message = "message sent";
+const newMember = "new member joined";
+const memberLeft = "member left server";
+const threadCreated = "thread created";
+const threadReplyRemoved = "thread reply removed";
+const threadTimeout = "thread timeout";
+const threadResolved = "thread resolved";
+
+export const threadStats = {
```

Comment:
Wouldn't it be safer to wrap this whole object in [`Object.freeze()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze) (or mark it with [TypeScript's `as const`](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions)) to prevent it from being modified (unless it is modified elsewhere) by any module, which imports it?

Reply:
Probably, but I'm not too worried about it